pkg/scalers/cpu_memory_scaler.go

```diff
@@ -54,11 +55,11 @@ func parseResourceMetadata(config *ScalerConfig) (*cpuMemoryMetadata, error) {
 		averageValueQuantity := resource.MustParse(value)
 		meta.AverageValue = &averageValueQuantity
 	case v2beta2.UtilizationMetricType:
-		valueNum, err := strconv.ParseInt(value, 10, 32)
+		valueNum, err := kedautil.ParseNumeric(value, 32)
 		if err != nil {
 			return nil, err
 		}
-		utilizationNum := int32(valueNum)
+		utilizationNum, _ := valueNum.(int32)
```

Comment:
it's doubtful that this type assertion would ever fail, but it would still be helpful to check it for success, as defensive programming:

```suggestion
		utilizationNum, ok := valueNum.(int32)
		if !ok {
		    return nil, fmt.Errorf("some error text")
		}
```